integration-tests/helpers/use-template-db.js

```diff
@@ -0,0 +1,104 @@
+const path = require("path")
+require("dotenv").config({ path: path.join(__dirname, "../.env") })
+
+const { createDatabase, dropDatabase } = require("pg-god")
+const { createConnection, getConnection } = require("typeorm")
+
+const DB_USERNAME = process.env.DB_USERNAME || "postgres"
+const DB_PASSWORD = process.env.DB_PASSWORD || ""
+const DB_URL = `postgres://${DB_USERNAME}:${DB_PASSWORD}@localhost`
+
+const pgGodCredentials = {
+  user: DB_USERNAME,
+  password: DB_PASSWORD,
+}
+
+class DatabaseFactory {
+  constructor() {
+    this.connection_ = null
+    this.masterConnectionName = "name"
+    this.templateDbName = "medusa-integration-template"
+  }
+
+  async createTemplateDb_() {
+    try {
+      const connection = await this.getMasterConnection()
+
+      const migrationDir = path.resolve(
```

Comment:
Could we in some way dynamically get the migrations directory so the tests will run in the same way if you are executing them from the root directory or from `integration-tests/api`. It might be a small thing but it would ensure that the tests could be executed in the same way as they have been previously.

I'm thinking something like this: 
```typescript
    const p = [process.cwd()]
      if (!p[0].endsWith("integration-tests/api")) {
        p.push("integration-tests/api")
      }

      const migrationDir = path.resolve(
        path.join(
          ...p,
          `node_modules`,
          `@medusajs`,
          `medusa`,
          `dist`,
          `migrations`
        )
      )
```